msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md

```diff
@@ -1,60 +1,134 @@
 ---
-title: Receive all channel messages with RSC
+title: Receive all conversation messages with RSC
 author: surbhigupta12
 description: Enable bots to receive all channel messages without being @mentioned using RSC permissions. Read on webApplicationInfo or authorization section in manifest.
 ms.topic: conceptual
 ms.localizationpriority: medium
 ---
 
-# Receive all channel messages with RSC
+# Receive all conversation messages with RSC
 
-The resource-specific consent (RSC) permissions model, originally developed for Teams Graph APIs, is extended to bot scenarios.
+The resource-specific consent (RSC) permissions model, originally developed for Teams Graph APIs, is being extended to bot scenarios.
 
-Using RSC, you can now request team owners to consent for a bot to receive user messages across standard channels in a team without being @mentioned. This capability is enabled by specifying the `ChannelMessage.Read.Group` permission in the manifest of an RSC enabled Teams app. After configuration, team owners can grant consent during the app installation process.
+Using RSC, you can request team owners consent for a bot to receive user messages across standard channels and group chats in a team without being @mentioned. The capability is enabled by specifying the `ChannelMessage.Read.Group` or `ChatMessage.Read.Chat` permissions in the manifest of a RSC enabled Teams app. The conversation owners can grant consent during the app installation or upgrade process. Bots can receive all group chat messages in a team when consented during app installation. For more information about enabling RSC for your app, see [resource-specific consent](../../../graph-api/rsc/resource-specific-consent.md).
```

Comment:
Some of the information in this paragraph is redundant with the latest edits. Please update with the content below:


With RSC, conversation owners can consent for a bot to receive all user messages in standard channels and chats without being @mentioned. This this can be enabled by specifying `ChannelMessage.Read.Group` or `ChatMessage.Read.Chat` permission strings in your Teams app manifest. Conversation owners can grant consent during the app installation or upgrade process after the app updates are published. For more information about enabling RSC for your app and inside of a tenant, see [resource-specific consent](../../../graph-api/rsc/resource-specific-consent.md).

Reply:
Updated as per suggestion.

Reply:
@v-abirade, the update that was made is different from what I shared and is not accurate. 

The statement below is not correct
`across standard channels and group chats in a team without being @mentioned`

Reply:
sorry, please ignore. Notice it was updated on the latest commit

Reply:
Thank you.